code/game/objects/structures/banking/banking.dm

```diff
@@ -0,0 +1,209 @@
+		user << "<span class = 'danger'>You have no hands to use this with.</span>"
+		return FALSE
+
+/datum/banking_program/atm
+    name = "Automatic Teller Machine"
+    description = "Connects to the FNB international digital banking service for quick withdrawal and deposit, as well as near-immediate response time."
+    var/depositing = FALSE
+
+
+
+/datum/banking_program/atm/do_html(mob/living/human/user)
+    if (mainmenu == "---")
+        mainmenu = "<h2>DIGITAL BANKING TERMINAL</h2><br>"
+        mainmenu += "<a href='?src=\ref[src];atm=2'>Deposit Cash</a>&nbsp;<a href='?src=\ref[src];atm=3'>Withdraw Cash</a>&nbsp;"
+
+/datum/banking_program/atm/Topic(href, href_list, hsrc)
+    mainbody = ""
+    if (href_list["atm"])
+        if (href_list["atm"] == 2)
+            if (user.bank_hold)
+                mainbody = "<font color ='red'><b>ACCESS DENIED</b></font> <br>"
+                mainbody += "<font color ='red'><b>DUE TO CERTAIN ISSUES, THIS ACCOUNT HAS BEEN PLACED ON HOLD UNTIL FURTHER NOTICE</b></font>"
+                playsound(loc, 'sound/machines/access_denied.ogg')
+                depositing = FALSE
+                return
+            else
+                mainbody += "Please insert the cash into the ATM. Only paper banknotes are accepted"
+                playsound(loc, 'sound/machines/atm_deposit_open_1.ogg', 100, TRUE)
+                depositing = TRUE
+                return
+
+        if (href_list["atm"] == 3)
+            mainbody += "Please select currency to withdraw, you have $[user.bank_dollar] and [user.bank_rubles] rubles"
+            mainbody += " <a href='?src=\ref[src];atm=wdollar'>(Dollar)</a> <a href='?src=\ref[src];atm=wrubles'>(Rubles)</a><br>
+
+        if(findtext(href_list["atm"], "wdollar"))
+            if (user.bank_hold)
+                mainbody += "<font color ='red'><b>ACCESS DENIED</b></font> <br>"
+                mainbody += "<font color ='red'><b>DUE TO CERTAIN ISSUES, THIS ACCOUNT HAS BEEN PLACED ON HOLD UNTIL FURTHER NOTICE</b></font>"
+                playsound(loc, 'sound/machines/access_denied.ogg')
+                sleep(0.5)
+                do_html(user)
+                return
+            else
+                mainbody = "Please input the amount you would like to withdraw <br>"
+                var/w_amount = input(usr, "How much in dollar would you like to withdraw?", "Amount", "50") as num|null
+                if ((user.bank_dollar - w_amount) < 0)
+                    mainbody = "<font color ='red'><b>ERROR: INSUFFICIENT FUNDS</b></font> <br>"
+                    sleep(0.5)
+                    do_html(user)
+                    return
+                else
+                    playsound(loc, "sound/machines/atm_withdraw.ogg")
+                    wait(32)
+                    user.bank_dollar -= w_amount
+                    new/obj/item/stack/money/dollar(loc, round(w_amount))
+                    mainbody = "<font color ='green'><b>SUCCESS! Money withdrawn</b></font> <br>"
```

Comment:
If you really wanted to, you could surround the "SUCCESS!" in <big> </big>

Reply:
infact, you should because this gives them the indicator they can leave, and it's bigger than the rest.